src/Cli/config.py

```diff
@@ -66,8 +66,27 @@ def remove_proxy():
     except Exception as e:
         click.echo(e)
 
+@click.command()
+@click.argument("user",type=str)
+def set_user(user):
+    try:
+        config = Config()
+        config.set_user(user)
+        click.echo(click.style('User information have been fetched.', fg='green'))
+    except Exception as e:
+        click.echo(click.style(e,fg='red'))
+@click.command()
+def get_user():
+    try:
+        config = Config()
+        click.echo(config.get_user())
```

Comment:
As asked in the last review as well, why are you directly printing the object. Can you consider formatting it?